tasks/elasticsearch-scripts.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-- set_fact: es_script_dir={{ es_conf_dir }}/{{es_instance_name}}
+- set_fact: es_script_dir={{ es_conf_dir }}{{ es_conf_dir }}{{'/' + es_instance_name if es_use_shared_fs else ''}}
```

Comment:
The double `es_conf_dir` looks like a bug?

Reply:
It does to me. I made a pull request with @bndabbs to remove it.

Reply:
This has been merged now.